src/citrine/resources/project.py

```diff
@@ -647,7 +647,11 @@ def search_all(self, search_params: Optional[Dict]) -> Iterable[Dict]:
 
         """
         collections = []
-        path = self._get_path(action="search")
+        if self.team_id is None:
+            path = "/projects/search"
+        else:
+            path = format_escaped_url("/teams/{team_id}/projects/search", team_id=self.team_id)
+
```

Comment:
I'd prefer to follow the existing patterns if possible, as opposed the the more local definition here.  Are there any routes that still should always be using a team-agnostic project access?  It seems like the better strategy here is to define `_path_template` as a `@property` that conditionally includes the team.  

I can code that up if it doesn't make sense.

Reply:
It totally makes sense. For this, we might need to do some research to verify that every endpoint called has a counterpart. If you agree, I would ticket that up, as I imagine I will find a few interesting things while doing it.
I can handle that myself after some high-priority work I need to address on another topic. What do you think?